ChangeLog

```diff
@@ -1,3 +1,428 @@
+  + libcrmcommon: deprecate RSC_STOP string constant
+  + libcrmcommon: deprecate RSC_STOPPED string constant
+  + libcrmcommon: deprecate shutdown_crm in enum action_tasks
+  + libcrmcommon: deprecate started_rsc in enum action_tasks
+  + libcrmcommon: deprecate start_rsc in enum action_tasks
+  + libcrmcommon: deprecate stonith_node in enum action_tasks
+  + libcrmcommon: deprecate stopped_rsc in enum action_tasks
+  + libcrmcommon: deprecate stop_rsc in enum action_tasks
+  + libcrmcommon: deprecate TYPE() macro
+  + libcrmcommon: deprecate XML_ATTR_VERBOSE string constant
+  + libcrmcommon: deprecate XML_CIB_ATTR_SOURCE string constant
+  + libcrmcommon: deprecate XML_CIB_TAG_DOMAINS string constant
+  + libcrmcommon: deprecate xml_has_children()
+  + libcrmcommon: deprecate XML_NODE_EXPECTED string constant
+  + libcrmcommon: deprecate XML_NODE_IN_CLUSTER string constant
+  + libcrmcommon: deprecate XML_NODE_IS_PEER string constant
+  + libcrmcommon: deprecate XML_NODE_JOIN_STATE string constant
+  + libcrmcommon: deprecate XML_RSC_ATTR_FAIL_STICKINESS string constant
+  + libcrmcommon: deprecate XML_RSC_ATTR_FAIL_TIMEOUT string constant
+  + libcrmcommon: deprecate XML_RSC_ATTR_INCARNATION_MAX string constant
+  + libcrmcommon: deprecate XML_RSC_ATTR_INCARNATION_MIN string constant
+  + libcrmcommon: deprecate XML_RSC_ATTR_INCARNATION_NODEMAX string constant
+  + libcrmcommon: deprecate XML_RSC_ATTR_PROMOTED_MAX string constant
+  + libcrmcommon: deprecate XML_RSC_ATTR_PROMOTED_NODEMAX string constant
+  + libcrmcommon: deprecate XML_TAG_DIFF_ADDED string constant
+  + libcrmcommon: deprecate XML_TAG_DIFF_REMOVED string constant
+  + libcrmcommon: deprecate XML_TAG_FRAGMENT
+  + libcrmcommon: dump_xml_formatted()'s argument is now const
+  + libcrmcommon: dump_xml_formatted_with_text()'s argument is const
+  + libcrmcommon: dump_xml_unformatted()'s argument is now const
+  + libcrmcommon: save_xml_to_file()'s xml argument is now const
+  + libcrmcommon: validate_xml_verbose()'s xml_blob argument is const
+  + libcrmcommon: write_xml_fd()'s xml argument is now const
+  + libcrmcommon: write_xml_file()'s xml argument is now const
+  + libcrmcommon: xml_top argument of xpath_search() is now const
+  + libcrmcommon,libpe_rules,libpe_status: move enum pe_ordering, struct
+    pe_action_wrapper_s, struct pe_tag_s, struct pe_ticket_s, struct
+    resource_object_functions_s, enum node_type, enum pe_action_flags, enum
+    pe_discover_e, enum pe_find, enum pe_link_state, enum pe_obj_types, enum
+    pe_quorum_policy, enum pe_restart, struct pe_action_s, struct pe_node_s,
+    struct pe_node_shared_s, struct pe_resource_s, struct pe_working_set_s,
+    enum action_fail_response, enum action_tasks, enum pe_print_options, enum
+    rsc_recovery_type, enum rsc_role_e, and enum rsc_start_requirement to
+    libcrmcommon
+  + libpacemaker,libpe_rules,libpe_status: use pcmk_action_t instead of pe_action_t
```

Comment:
You'll want a comma at the end of this line

Reply:
thanks